misc/roundtrip

```diff
@@ -125,24 +125,22 @@ IFS='
 '
 
 tagfiles=$(find "$UNITS" -name expected.tags)
-for tags in $tagfiles; do
-    tagnames=$(sed -e 's/^\([^	]*\)	.*/\1/' "$tags" | limit_entries)
-    for name in $tagnames; do
-	# Yes, there is a reason for this craziness.  We need to properly
-	# handle embedded newlines (expanded from "\n"), including trailing
-	# ones the shell would strip automatically.  To work around this, we
-	# add a dummy character at the end to inhibit stripping, and then
-	# remove it, plus the extra newline, using variable substitutions.
-	# Note: we use "printf '%s\n'" instead of "echo" because Dash's "echo"
-	# unconditionally expands some sequences, like "\t" and alike.
-	t="$(printf '%s\n' "$name" | expandEscapeSequences; printf _)"
-	t="${t%
-_}"
-	if [ 1 -gt $("${READTAGS}" -t "$tags" - "$t" | wc -l) ]; then
-	    printf 'FAILED: "%s" -t "%s" - "%s"\n' "${READTAGS}" "$tags" "$t"
-	    printf '	The raw tag name was "%s"\n' "$name"
-	    s=1
-	fi
+for tagfile in $tagfiles; do
+    for tagname in $(cut -f1 < "$tagfile" | limit_entries)
+    do
```

Comment:
Instead of 
```
for ...
do
```
, could you keep
```
for ...; do
```

The area we are dealing with is quite complicated.  So let's minimize the diff.